rclpy/rclpy/timer.py

```diff
@@ -80,3 +84,65 @@ def time_since_last_call(self):
     def time_until_next_call(self):
         with self.handle as capsule:
             return _rclpy.rclpy_time_until_next_call(capsule)
+
+
+class Rate:
```

Comment:
Consider moving the implementation to it's own module `rate.py`.

Reply:
I thought about it, but I think it fits in this module since it is a wrapper around a timer. It matches the [`Rate` class location in ROS 1](https://github.com/ros/ros_comm/blob/6b1ee1a43b5b0a970d85612435e269818c89a152/clients/rospy/src/rospy/timer.py#L47) too.

Reply:
Fair enough.